### ISSUE TITLE
[bounty]  Speech Recognition: Add sherpa-onnx-sense-voice support for Chinese

### ISSUE THREAD

**Issue by brucel11qwe** [mediar-ai/screenpipe#548]
"The effect of Whisper in Chinese speech recognition is very poor, with almost all recognitions being incorrect. I hope to add support for sherpa-onnx-sense-voice-zh-en-ja-ko-yue-2024-07-17."
This translation conveys your concern about the poor performance of Whisper for Chinese speech recognition and your desire to incorporate a specific model (sherpa-onnx-sense-voice-zh-en-ja-ko-yue-2024-07-17) for better results.

**Comment by louis030195** [comment]
@brucel11qwe but would love a contribution for this model!

did you try using Chinese as language in the settings?

**Comment by brucel11qwe** [comment]
Yes, I have configured the settings and I'm using the openai/whisper-large-v3-turbo   model. However, Whisper's support for Chinese language is quite poor, with almost all recognition results being incorrect. I suggest you try https://github.com/FunAudioLLM/SenseVoice - its performance is about 100 times better than Whisper for Chinese speech recognition

**Comment by louis030195** [comment]
> Yes, I have configured the settings and I'm using the openai/whisper-large-v3-turbo model. However, Whisper's support for Chinese language is quite poor, with almost all recognition results being incorrect. I suggest you try https://github.com/FunAudioLLM/SenseVoice - its performance is about 100 times better than Whisper for Chinese speech recognition

can you send a PR for it?

you can use ONNX runtime that they support - check example we use for VAD to use ONNX in rust:

https://github.com/thewh1teagle/vad-rs/tree/main

🙏

**Comment by louis030195** [comment]
screenpipe 2.2.0 is here! 🎉 https://x.com/louis030195/status/2014403576475615312

closing due to inactivity. @brucel11qwe speech recognition has been updated - please try the new version!
